.gitignore

```diff
@@ -6,3 +6,5 @@
 .vscode/
 .idea/
 **/*~
+
+tmp/
```

Comment:
I often have some temporary files from other branches experiments that I don't want to lose and don't want to commit to the current branch. I suppose it's ok to have a `tmp/` in gitignore in genera.